clients/admin-ui/src/features/consent-settings/GppConfiguration.tsx

```diff
@@ -0,0 +1,111 @@
+import { Divider, Stack, Text } from "@fidesui/react";
+import { useFormikContext } from "formik";
+import { ReactNode } from "react";
+
+import { useAppSelector } from "~/app/hooks";
+import { useFeatures } from "~/features/common/features";
+import {
+  CustomCheckbox,
+  CustomRadioGroup,
+  CustomSwitch,
+} from "~/features/common/form/inputs";
+import { selectGppSettings } from "~/features/privacy-requests";
+import { GPPSettings, GPPUSApproach } from "~/types/api";
+
+import FrameworkStatus from "./FrameworkStatus";
+import SettingsBox from "./SettingsBox";
+
+const Section = ({
+  title,
+  children,
+}: {
+  title: string;
+  children: ReactNode;
+}) => (
+  <Stack spacing={3} mb={3} data-testid={`section-${title}`}>
+    <Text fontSize="sm" fontWeight="bold" lineHeight={5} color="gray.700">
+      {title}
+    </Text>
+    {children}
+  </Stack>
+);
+
+const GppConfiguration = () => {
+  const { tcf: isTcfEnabled } = useFeatures();
+  const gppSettings = useAppSelector(selectGppSettings);
+  const isEnabled = !!gppSettings.enabled;
+  const { values } = useFormikContext<{ gpp: GPPSettings }>();
+  const showMspa = !!values.gpp.us_approach;
+
+  return (
+    <SettingsBox title="Global Privacy Platform">
+      <Stack spacing={6}>
+        <FrameworkStatus name="GPP" enabled={isEnabled} />
+        {isEnabled ? (
+          <>
+            <Section title="GPP U.S.">
+              <CustomRadioGroup
+                name="gpp.us_approach"
+                variant="stacked"
+                defaultFirstSelected={false}
+                options={[
+                  {
+                    label: "Enable U.S. National",
+                    value: GPPUSApproach.NATIONAL,
+                    tooltip: "TODO",
```

Comment:
Just want to double-check because this is totally the kind of thing I would do early in my work and then forget to double back for before merging-- are proper tooltips for these being worked on?

Reply:
ah yes thanks for the reminder!! will ping @mfbrown for some tooltip text

Reply:
okay added tooltips!


https://github.com/ethyca/fides/assets/24641006/70702b9d-4c70-4a0f-8f39-9ad54afafb3c